docs/spec/openapi.yaml

```diff
@@ -0,0 +1,16368 @@
+                        allow: true
+                      password_protected: false
+                      url: ""
+                    name: dir
+                    path_lower: /dir
+                    permissions: []
+                    policy:
+                      acl_update_policy:
+                        ".tag": owner
+                      member_policy:
+                        ".tag": anyone
+                      resolved_member_policy:
+                        ".tag": team
+                      shared_link_policy:
+                        ".tag": anyone
+                    preview_url: https://www.dropbox.com/scl/fo/fir9vjelf
+                    shared_folder_id: "84528192421"
+                    time_invited: 2016-01-20T00:00:00Z
+  /sharing/share_folder:
+    post:
+      tags:
+      - sharing
+      summary: share_folder
+      description: |-
+        [share_folder](https://www.dropbox.com/developers/documentation/http/documentation#sharing-share_folder)
+
+        scope: `sharing.write`
+
+        Share a folder with collaborators.
+        Most sharing will be completed synchronously. Large folders will be completed asynchronously. To make testing the async case repeatable, set `ShareFolderArg.force_async`.
+        If a `ShareFolderLaunch.async_job_id` is returned, you'll need to call `check_share_job_status` until the action completes to get the metadata for the folder.
+      operationId: shareFolder
+      parameters:
+      - $ref: '#/components/parameters/DropboxApiPathRoot'
+      - $ref: '#/components/parameters/DropboxApiSelectUser'
+      - $ref: '#/components/parameters/DropboxApiSelectAdmin'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/sharing_share_folder_body'
+            examples:
+              share_folder:
+                value:
+                  access_inheritance: inherit
+                  acl_update_policy: editors
+                  force_async: false
+                  member_policy: team
+                  path: /example/workspace
+                  shared_link_policy: members
+      responses:
+        "200":
+          description: share_folder response
+          headers:
+            X-Dropbox-Request-Id:
+              style: simple
+              explode: false
+              schema:
+                type: string
+                example: "1234"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/inline_response_200_33'
+              examples:
+                share_folder response:
+                  value:
+                    ".tag": complete
+                    access_inheritance:
+                      ".tag": inherit
+                    access_type:
+                      ".tag": owner
+                    is_inside_team_folder: false
+                    is_team_folder: false
+                    link_me
```

Comment:
it’s generally recommended to avoid using verbs as prefixes in record names. The best practice is to focus on the **state or data** the record represents rather than the action performed on it.
Shall we simplify the object name by minimizing action verb for clarity and Ballerina naming alignment?

ex: AddPropertiesRequest  -> NewPropertiesRequest, PropertiesRequest

Reply:
Please check and apply this to other relevant places.